components/AddButton.js

```diff
@@ -3,19 +3,26 @@ import PropTypes from 'prop-types';
 import Button from '@material-ui/core/Button';
 
 const AddButton = props => {
```

Comment:
The following would be more idiomatic, rather than having the separate line for destructuring. It would also allow you to have an implicit return on the function.
```suggestion
const AddButton = ({ text, action }) => (
```

Reply:
You'll need to resolve this locally rather than accepting the suggestion to get the implicit return and no errors

Reply:
Much better. Thanks. Will make the change.